clickhouse-jdbc/src/test/java/com/clickhouse/jdbc/AccessManagementTest.java

```diff
@@ -106,4 +102,80 @@ private String getServerVersion(Connection connection) {
         }
         return null;
     }
+
+    @Test
+    public void testSetRolesAccessingTableRows() throws SQLException {
+        String httpEndpoint = "http://" + getServerAddress(ClickHouseProtocol.HTTP) + "/";
+        String url = String.format("jdbc:ch:%s", httpEndpoint);
+        Properties properties = new Properties();
+        properties.setProperty(ClickHouseHttpOption.REMEMBER_LAST_SET_ROLES.getKey(), "true");
+        properties.setProperty(ClickHouseHttpOption.CONNECTION_PROVIDER.getKey(),
+                HttpConnectionProvider.APACHE_HTTP_CLIENT.name());

```

Comment:
Why we are setting this, it is the default

Reply:
I've forgot this after testing. Will remove.